test/jdk/java/util/UUID/UUIDTest.java

```diff
@@ -241,6 +327,12 @@ private static void variantTest() throws Exception {
             throw new Exception("nameUUIDFromBytes not variant 2");
         }
 
+        long now = System.currentTimeMillis();
+        test = UUID.epochMillis(now);
+        if (test.variant() != 2) {
+            throw new Exception("timestampUUID not variant 2");
+        }
+
```

Comment:
Can this be moved/be merged with the test at line 270-274.
Generally, put the new tests for UUID.epochMillis together.